### ISSUE TITLE
Cannot find minecraft_ff_in.jar

### ISSUE THREAD

**Issue by jLynx** [ThisTestUser/MCPFixer#2]
Hi, great work with this project.
One issue I am having is that its failing when trying to find or create ```minecraft_ff_in.jar```
Below is the full log

```
== MCP 9.40 (data: 9.41, client: 1.13, server: 1.13) ==
"scalac" is not found on the PATH.  Scala files will not be recompiled
# found ff, ff patches, srgs, name csvs, doc csvs, param csvs, astyle, astyle co
nfig, rg, ss
!! Modified jar detected. Unpredictable results !!
== Decompiling client using fernflower ==
> Creating SRGs
> Applying SpecialSource
> Applying MCInjector
> Creating renamed srg
> Filtering classes
> Decompiling
FATAL ERROR
Traceback (most recent call last):
  File "runtime\decompile.py", line 145, in decompile joined=only_joined)
  File "A:\Users\Joel\Documents\Code\Minecraft\MCP Fixer\mcp940\runtime\mcp.py", line 47, in decompile_side commands.applyff(side)
  File "A:\Users\Joel\Documents\Code\Minecraft\MCP Fixer\mcp940\runtime\commands.py", line 1402, in applyff shutil.move(jarout, ffoutput[side])
  File "shutil.pyc", line 299, in move
  File "shutil.pyc", line 128, in copy2
  File "shutil.pyc", line 82, in copyfile IOError: [Errno 2] No such file or directory: 'temp\\src\\minecraft\\minecraft_ff_in.jar'
Press any key to continue . . .
```

After it breaks, this is my folder structure in ```mcp940>temp```:

![image](https://user-images.githubusercontent.com/4393979/60411353-91c4ef80-9c20-11e9-8348-e389aa87b74d.png)

The ```mcp940>temp>src>minecraft``` folder is there but empty

Do you have any ideas why this is happening or what I need to do to fix it?

**Comment by ThisTestUser** [comment]
Please upload your log at logs/mcp.log. The log you gave me doesn't really show anything.

**Comment by jLynx** [comment]
Hi @ThisTestUser 
Here is a fresh log from my latest run [mcp.log](https://github.com/ThisTestUser/MCPFixer/files/3351856/mcp.log)

**Comment by ThisTestUser** [comment]
Can you take a screenshot of your conf folder? It looks like you're missing something there.
EDIT: I think I made a mistake in the wiki. Search for CmdExceptor in the wiki, and replace those two lines again.

**Comment by jLynx** [comment]
will try thst in a few minutes,but in the meantime, here is that screenshot 
![image](https://user-images.githubusercontent.com/4393979/60540453-4cf5a180-9d63-11e9-97aa-2a13d4db622b.png)

**Comment by ThisTestUser** [comment]
You should also replace the .csv files with the 1.13 version available here: http://export.mcpbot.bspk.rs/

**Comment by jLynx** [comment]
Also a side note, in the wiki on the first paragraph where you say to replace the files in ```mcp940/conf/mcp.cfg``` it should be ```mcp940/conf/version.cfg```

**Comment by jLynx** [comment]
It got further this time, but it failed again with a new error:

Here is the link to the log
[mcp.log](https://github.com/ThisTestUser/MCPFixer/files/3352080/mcp.log)


```
== MCP 9.40 (data: 9.41, client: 1.13, server: 1.13) ==
"scalac" is not found on the PATH.  Scala files will not be recompiled
# found ff, ff patches, srgs, name csvs, doc csvs, param csvs, astyle, astyle co
nfig, rg, ss
!! Modified jar detected. Unpredictable results !!
== Decompiling client using fernflower ==
> Creating SRGs
> Applying SpecialSource
> Applying MCInjector
> Creating renamed srg
> Filtering classes
> Decompiling
FATAL ERROR
Traceback (most recent call last):
  File "runtime\decompile.py", line 145, in decompile
    joined=only_joined)
  File "A:\Users\Joel\Documents\Code\Minecraft\MCP Fixer\mcp940\runtime\mcp.py",
 line 47, in decompile_side
    commands.applyff(side)
  File "A:\Users\Joel\Documents\Code\Minecraft\MCP Fixer\mcp940\runtime\commands
.py", line 1402, in applyff
    shutil.movse(jarout, ffoutput[side])
AttributeError: 'module' object has no attribute 'movse'
Press any key to continue . . .
```

**Comment by jLynx** [comment]
So it looks like that should be move instead of movse?

Edit, that could have just been a typo from me..oops. Time to try again!

**Comment by jLynx** [comment]
Ok, pretty much done. 
The only error I got now is: `!! Missing server jar file. Aborting !!`
Everything else seems to work. Is that a problem?

To sum up. 

- For some stupid reason I forgot to replace the .csv's with the correct ones.
- The code on the Wiki may have not been correct

So it all seems to be working now for 1.13.
Thanks for the help!

_(Time to try run it. Then try with 1.14.3 with all the stuff I learned here)_
